src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -67,20 +70,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   /** Stores a new {@link Message}. */
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException
+  {
 
     UserService userService = UserServiceFactory.getUserService();
-    if (!userService.isUserLoggedIn()) {
+    if (!userService.isUserLoggedIn())
+    {
       response.sendRedirect("/index.html");
       return;
     }
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String recipient = request.getParameter("recipient"); //added recipient variable
 
-    Message message = new Message(user, text);
+    Message message = new Message(user, text, recipient); //added recipient as a parameter
     datastore.storeMessage(message);
 
-    response.sendRedirect("/user-page.html?user=" + user);
+    response.sendRedirect("/user-page.html?user=" + recipient); //changed user to recipient so it takes user back to previous page
```

Comment:
This comment sounds more like what your pull request title should be. Please remove this comment from code and put this as your pull request title.

Reply:
I have removed this along with all other comments. I'll take note of this for future pull requests